server/data/hmppsAuthClient.ts

```diff
@@ -19,10 +17,10 @@ function getSystemClientTokenFromHmppsAuth(username?: string): Promise<superagen
 
   const grantRequest = new URLSearchParams({
     grant_type: 'client_credentials',
-    ...(username && { username }),
+    username, // todo
```

Comment:
Is there something still `todo` here?

Reply:
Not in this PR but in the overall story yes - need to make sure we're passing through all of the correct information to the backend and wanted to leave myself a marker so I'd remember where it needed to be changed.